docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -93,6 +97,38 @@ To add a tag, follow these steps:
 1. Select a value from the **Select value** drop-down. This field is populated based upon the tag.
 1. Optional: Select **+** to add an additional tag.
 
+## Optional: Use Aggregate by
+
+{{% admonition type="warning" %}}
+**Aggregate by** is an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided.
+[Enable the `metricsSummary` feature toggle](/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) in Grafana to use this feature. Your Grafana Tempo data source must also point to a Tempo database with the [Metrics Summary API](https://grafana.com/docs/tempo/latest/api_docs/metrics-summary/) enabled. Contact Grafana Support to enable this feature in Grafana Cloud.
+{{% /admonition %}}
+
+Using **Aggregate by**, you can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` that match your filter criteria, grouped by one or more attributes.
+This capability is based on the [metrics summary API](/docs/grafana-cloud/monitor-infrastructure/traces/metrics-summary-api/).
+
+When you use **Aggregate by**, the selections you make determine how the information is reported in the Table. Every combination that matches selections in your data is listed in the table.
+Each aggregate value, for example `intrinsic`:`name`, has a corresponding column in the results table.
+
+For example, **names** matching `GET /:endpoint` with a **span.http.user_agent** of `k6/0.46` appeared in 31,466 spans. Instead of being listed by traces and associated spans, the query results are grouped by the the selections in **Aggregate by**.
+
+The RED metrics are calculated for every name and user agent combination found in your data.
+
+![Use Aggregate by to calculate RED metrics for spans and group by attributes](/static/img/docs/tempo/screenshot-traces-aggregate-by.png)
+
+The screenshot shows all of the successful HTTP `status_code` API calls against the `mystical-server` service.
+The results are shown in the same order used in **Aggregate by**.
+For example, **Aggregate by** lists `intrinsic.name` followed by `span.http.user_agent`.
+The first column in the results Table shows **name** and then **span.http.user_agent**.
+
+To use this capability:
+
+1. In the **Aggregate by** row, select a scope from the first drop-down box. For example, `span`.
+1. Select an attribute from the second drop-down.
+1. Optional: Select **+** to add an **Aggregate by** row.
+1. Optional: Select a **Time range** to expand or narrow the data set.
```

Comment:
Note: the metrics summary only calculates summaries based on the last hour. So it does use the time range, but only up to an hours range.

Reply:
Thank you! I will add a note. 
```suggestion
1. Optional: Select a **Time range** to expand or narrow the data set. Data is available for the last hour.
```

Reply:
@joey-grafana -- What happens if I select a time range outside of the last hour? Do I get an error? Or do we still return results but it is just over the last hour of data?

Reply:
Ideally we would return an error that says something along the lines of: "sorry you requested an invalid time range. try again with a time range sometime in the last 1 hour".